php/docker/templates/nginx-conf.d/90-metrics.conf

```diff
@@ -60,32 +111,114 @@ server {
 
     location {{ default "/metrics/nginx" .Env.STACK_METRICS_NGINX_PATH }} {
         content_by_lua_block {
-            metric_connections:set(
+            metrics_nginx["http_connections"]:set(
                 ngx.var.connections_reading, {
                     ngx.var.server_name,
                     {{ $siteName }},
                     ngx.var.host,
                     "reading"
                 }
             )
-            metric_connections:set(
+            metrics_nginx["http_connections"]:set(
                 ngx.var.connections_waiting, {
                     ngx.var.server_name,
                     {{ $siteName }},
                     ngx.var.host,
                     "waiting"
                 }
             )
-            metric_connections:set(
+            metrics_nginx["http_connections"]:set(
                 ngx.var.connections_writing, {
                     ngx.var.server_name,
                     {{ $siteName }},
                     ngx.var.host,
                     "writing"
                 }
             )
-            prometheus:collect()
+            prometheus_nginx:collect()
+        }
+    }
+
+    location {{ default "/metrics/php-fpm" .Env.STACK_METRICS_PHP_PATH }} {
+    {{- if ne "off" ( default "/-/php-status" .Env.PHP_STATUS_PATH ) }}
+        content_by_lua_block {
+            local httpc = http.new()
+
+            local stats_url = "http://127.0.0.1:"
+              .. {{ default "8080" .Env.PORT | quote }}
+              .. {{ default "/-/php-status" .Env.PHP_STATUS_PATH | quote }}
+              .. "?json"
+
+            local res, err = httpc:request_uri(stats_url, { method = "GET" })
+
+            if not res then
```

Comment:
You should also check for 200 status code or what is expected in this situation. Err in this case only represents a connection error, malformed uri etc.

Reply:
`res` is `nil` if the response status is not `20x`, and if successfull it will always have a `.body`, so that test should be enough.
https://github.com/ledgetech/lua-resty-http#synopsis  
https://github.com/ledgetech/lua-resty-http#request_uri

Reply:
> res is nil if the response status is not 20x
I didn't see where it says that in the provided links. Also I don't see how that could be the case, since you won't be able to verify other status codes.